packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -30,37 +26,20 @@ const stubCreateEthers = (ctrl: TokensController, res: boolean) => {
 describe('TokensController', () => {
   let tokensController: TokensController;
   let preferences: PreferencesController;
-  let network: NetworkController;
-  let messenger: NetworkControllerMessenger;
 
   let instEthProvStub: sinon.SinonStub;
+  let onNetworkStateChangeCallback: any;
+
+  const rinkeby = { chainId: '4', type: 'rinkeby' } as ProviderConfig;
+  const ropsten = { chainId: '3', type: 'ropsten' } as ProviderConfig;
 
   beforeEach(() => {
-    messenger = new ControllerMessenger().getRestricted({
-      name: 'NetworkController',
-      allowedEvents: [
-        'NetworkController:stateChange',
-        'NetworkController:providerConfigChange',
-      ],
-      allowedActions: [],
-    });
     preferences = new PreferencesController();
-    network = new NetworkController({
-      messenger,
-      infuraProjectId: 'potato',
-    });
-
     tokensController = new TokensController({
       onPreferencesStateChange: (listener) => preferences.subscribe(listener),
       onNetworkStateChange: (listener) =>
-        messenger.subscribe('NetworkController:stateChange', (state, patch) => {
-          const touchesProviderConfig = patch.find(
-            (p) => p.path[0] === 'providerConfig',
-          );
-          if (touchesProviderConfig) {
-            listener(state);
-          }
-        }),
+        (onNetworkStateChangeCallback = (providerConfig: ProviderConfig) =>
```

Comment:
It seems like all we are doing here is wrapping `providerConfig` in an object. If that's the case, I'm wondering if it would be simpler to capture the `listener` itself rather than wrapping it? It seems like that might be easier to understand as well.

It also seems a bit odd that the name of this function is a noun — usually names of functions are verbs. What if we call this `changeNetworkState` or `issueNetworkStateChange` or something like that?